python/py-pyside2/Portfile

```diff
@@ -31,6 +31,10 @@ if {${name} ne ${subport}} {
     # see https://trac.macports.org/ticket/57517
     qt5.min_version     5.8
 
+    # Needed for generating shiboken2 documentation
+    qt5.depends_build_component \
+        sqlite-plugin
```

Comment:
Error observed if this dependency isn't specified:
```
Genereting QCH based on the QHP...
QSqlDatabase: QSQLITE driver not loaded
QSqlDatabase: available drivers:
Cannot open data base file /opt/local/var/macports/build/_opt_local_var_macports_sources_github.com_macports_macports-ports_python_py-pyside2/py38-pyside2/work/pyside-setup-opensource-src-5.15.1/pyside3_build/py3.8-qt5.14.2-64bit-release/shiboken2/doc/html/Shiboken.qch.
Building up file structure...
make[3]: *** [doc] Error 1
make[2]: *** [doc/CMakeFiles/doc.dir/all] Error 2
make[1]: *** [doc/CMakeFiles/doc.dir/rule] Error 2
make: *** [doc] Error 2
error: Error generating documentation for shiboken2
***** Exception while trying to prepare bdist_wheel override class: cannot import name 'pep425tags' from 'wheel' (/opt/local/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/wheel/__init__.py). Skipping wheel overriding.
Traceback (most recent call last):
  File "setup.py", line 296, in <module>
    setup_runner.run_setup()
  File "/opt/local/var/macports/build/_opt_local_var_macports_sources_github.com_macports_macports-ports_python_py-pyside2/py38-pyside2/work/pyside-setup-opensource-src-5.15.1/build_scripts/setup_runner.py", line 162, in run_setup
    raise RuntimeError(msg)
RuntimeError:
setup.py invocation failed with exit code: 1.


setup.py invocation was: /opt/local/Library/Frameworks/Python.framework/Versions/3.8/bin/python3.8 setup.py --no-user-cfg build -j2 --qmake=/opt/local/libexec/qt5/bin/qmake --cmake=/opt/local/bin/cmake --parallel=2 --macos-deployment-target=10.15 --macos-sysroot=/Library/Developer/CommandLineTools/SDKs/MacOSX10.15.sdk --internal-build-type=shiboken2

Command failed:  cd "/opt/local/var/macports/build/_opt_local_var_macports_sources_github.com_macports_macports-ports_python_py-pyside2/py38-pyside2/work/pyside-setup-opensource-src-5.15.1" && /opt/local/Library/Frameworks/Python.framework/Versions/3.8/bin/python3.8 setup.py --no-user-cfg build -j2 --qmake=/opt/local/libexec/qt5/bin/qmake --cmake=/opt/local/bin/cmake --parallel=2 --macos-deployment-target=10.15 --macos-sysroot=/Library/Developer/CommandLineTools/SDKs/MacOSX10.15.sdk
Exit code: 1
```

This dependency was not needed by pyside2 5.15.0.